sampleconfig/sample-dcrctl.conf

```diff
@@ -37,3 +33,21 @@
 
 ; RPC server certificate chain file for validation
 ; rpccert=~/.dcrd/rpc.cert
+
+; Disable TLS
+; notls=0
```

Comment:
It looks like all the existing boolean options in this config (testnet=1, simnet=1) set the non-default value in the commented out sample line, so these three (notls, skipverify, and json) should probably set =1 as well?  This also matches how dcrd's sample config reads.